### ISSUE TITLE
[BUG] in function error.

### ISSUE THREAD

**Issue by chriztheanvill** [rmagatti/auto-session#101]
**Describe the bug**
Shows an error after load session.

**To Reproduce**
1. Save session.
2. Load Session saved.

**Expected behavior**
Load session normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/37992958/144289793-c530f8dc-4617-4032-a8ab-1547ffad5864.png)

**Baseline (please complete the following information):**
 - OS. e.g `uname -a`: Linux cris-pc 5.13.0-22-generic #22-Ubuntu SMP Fri Nov 5 13:21:36 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
 - Neovim version `nvim --version` : v0.6.0
 - URL to your current config (if public)

**Additional context**

**Comment by rmagatti** [comment]
Hey, thanks for the submission.

This is actually expected behaviour. Your config loading is throwing that Fold not found error upoon doing the `source yoursession.vim` command.
When auto-session encounters an error while trying to source it prints this out and disables auto-save to avoid further saving of the session with potentially stuff you wouldn't want after an error loading yor config.

TL;DR: this is something with your config, gotta find whatever is throwing that error and fix that. auto-session should work find after you do that.

Let me know if this doesn't do it for you or if you would have expected different behaviour.

**Comment by chriztheanvill** [comment]
I found a problem.
When the diagnostics popup shows and save the session, then load the session, this error occurs.

Also, as you said, when I use fold, show this error.

**Comment by rmagatti** [comment]
Hey @chriztheanvill, so there doesn't seem to be anything wrong with auto-session then? Just those config errors you mentioned?
